.github/CODEOWNERS

```diff
@@ -54,4 +54,11 @@
 /Code/Tools/AssetBundler/ @o3de/sig-content-maintainers
 /Code/Tools/AssetProcessor/ @o3de/sig-content-maintainers
 /Code/Tools/SceneAPI/ @o3de/sig-content-maintainers
-/Gems/SceneProcessing/ @o3de/sig-content-maintainers
\ No newline at end of file
+/Gems/SceneProcessing/ @o3de/sig-content-maintainers
+
+# SIG-Build areas of ownership
+/scripts/build @o3de/sig-build-reviewers @o3de/sig-build-maintainers
+/scripts/commit_validation @o3de/sig-build-maintainers
+/scripts/license_scanner @o3de/sig-build-maintainers
+/scripts/signer @o3de/sig-build-maintainers
```

Comment:
Any areas of Code in the cmake folder that the build team wants to add to their areas of ownership?

Reply:
Yes, we discussed it earlier with the SIG to do a second pass, but I think we can at least take oversight of the installer scripts in `cmake/`

Reply:
It should be fine to have overlapping areas of ownership for certain areas of the codebase.